script/Common/00_02.txt

```diff
@@ -406,7 +406,7 @@ The sounds of his footsteps are gradually approaching.
 [sha:b1037616e408e4c319ec36fba2b6207c3be1d99f]{
 -- Page 18, Offset 91.
 -- なのにボクはぼんやりと
-And yet, somehow it all feels vague.
+And yet, I'm still in a daze.
```

Comment:
Yeah, I couldn't come up with what I felt was a nice way of connecting them. Maybe

> And yet, it doesn't register to me―――
> All I do is stare blankly into the azure haze above.

Reply:
That sounds pretty nice.
I think you should probably keep it to just blue though. Simple vocabulary for the lost kid

Reply:
I kinda like azure here because despite this being a kid he uses what seems to be a way less common kanji here - 青い is like #118 for frequency compared to #2285 for 碧い

Reply:
Yeah, but Nasu is cheating and adding uncommon kanjis to normal words (when spoken out loud). I dunno, I'd rather be more all-in on the nature of the scene. The Japanese can have both because haha あお can be the blue everyone know but also spelled weird because its Japanese.